src/initial_trees.py

```diff
@@ -53,7 +60,7 @@ def GenerateInitialTrees(self) -> list:
         for tree_number in range(self.NUM_INIT_TREES):
             command = "cat tree." + str(tree_number) + ".treefile >> parsimony.treefile"
             os.system(command)
-            initial_tree = ReadFile(f'tree.{tree_number}.treefile')
+            initial_tree = ReadRandomLine(f'tree.{tree_number}.treefile')
```

Comment:
You want to keep MPBoot's functionality here, so put the 'ReadFile' back and add an 'if, else'. This way you are adding functionality, rather than replacing it.